src/main/java/org/isf/medicalinventory/gui/InventoryWardEdit.java

```diff
@@ -689,10 +1131,40 @@ public boolean isCellEditable(int rowIndex, int columnIndex) {
         }
     }
 
-    private List<MedicalInventoryRow> loadNewInventoryTable(String code) throws OHServiceException {
+    private List<MedicalInventoryRow> loadNewInventoryTable(String code, MedicalInventory inventory, boolean add) throws OHServiceException {
+        List<MedicalInventoryRow> inventoryRowsList;
+        if (inventory != null) {
+            int id = inventory.getId();
+            inventoryRowsList = medicalInventoryRowManager.getMedicalInventoryRowByInventoryId(id);
+            if (add) {
+                inventoryRowsList = getMedicalInventoryRows(code);
+            }
+        } else {
+            inventoryRowsList = getMedicalInventoryRows(code);
+        }
+        return inventoryRowsList;
+    }
+
+    private void adjustWidth() {
+        for (int i = 0; i < jTableInventoryRow.getColumnModel().getColumnCount(); i++) {
+            jTableInventoryRow.getColumnModel().getColumn(i).setPreferredWidth(pColumwidth[i]);
+            if (i == 0 || !pColumnVisible[i]) {
+                jTableInventoryRow.getColumnModel().getColumn(i).setMinWidth(0);
+                jTableInventoryRow.getColumnModel().getColumn(i).setMaxWidth(0);
+                jTableInventoryRow.getColumnModel().getColumn(i).setPreferredWidth(0);
+            }
+        }
+        DefaultTableCellRenderer centerRenderer = new DefaultTableCellRenderer();
+        centerRenderer.setHorizontalAlignment(JLabel.CENTER);
```

Comment:
```suggestion
        centerRenderer.setHorizontalAlignment(SwingConstants.CENTER);
```
Should use the static base class and not accessed via derived types